pom.xml

```diff
@@ -55,8 +55,8 @@
                 <artifactId>maven-compiler-plugin</artifactId>
                 <version>3.8.1</version>
                 <configuration>
-                    <source>9</source>
-                    <target>9</target>
+                    <source>17</source>
```

Comment:
Library cannot depend on newer JDK versions since it'll prevent users with older versions from using this library